.github/workflows/staging.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: RIF Marketplace PINNER deploy
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-**'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - staging
+      - features/devops
+
+jobs:
+  deploy-staging:
```

Comment:
Also, as in the UI PR, will we have jobs for the testnet and mainnet?

Reply:
And what is the process of adding job for the develop branch? We can add it only after we have it set up in the infra?

Reply:
"Also, as in the UI PR, will we have jobs for the testnet and mainnet?" : We will have workflows for every environment, I'm working doing only staging CD to test and validate workflows that can later be propagated to the other Envs easily 
"And what is the process of adding job for the develop branch? We can add it only after we have it set up in the infra?" Yes each branch corresponds to an infra environment, so develop will be deployed on develop infrastructure, staging on staging infrastructure and so on, s we still don't have dev env we cannot deploy it yet.